docs/topics/leaks.rst

```diff
@@ -61,8 +61,8 @@ Debugging memory leaks with ``trackref``
 ========================================
 
 :mod:`trackref` is a module provided by Scrapy to debug the most common cases of
-memory leaks. It basically tracks the references to all live Requests,
-Responses, Item and Selector objects.
+memory leaks. It basically tracks the references to all live Request,
+Response, BaseItem, Spider and Selector objects.
```

Comment:
Given #4534 I guess we need to review the new usages of `BaseItem`, and in cases like these revert them to `Item`.